src/sidebar/components/LoggedOutMessage.js

```diff
@@ -18,36 +18,32 @@ function LoggedOutMessage({ onLogin }) {
   const store = useStoreProxy();
 
   return (
-    <div className="LoggedOutMessage">
-      <span>
+    <div className="flex flex-col items-center m-6 space-y-6">
+      <span className="text-center">
         This is a public annotation created with Hypothesis. <br />
         To reply or make your own annotations on this document,{' '}
-        <a
-          className="LoggedOutMessage__link"
+        <Link
+          classes="inline text-color-text underline hover:underline"
```

Comment:
YALDP (Yet Another Link Design Pattern): We style our links so many different ways... I'll revisit these after I finish this pass...(I'll be revisiting all shared component use with `classes`).